Runtime/Data/Collection/ObservableCounter.cs

```diff
@@ -45,14 +45,13 @@ public virtual void IncreaseCount(TElement element)
                 return;
             }
 
-            if (!ElementsCounter.ContainsKey(element))
+            if (ElementsCounter.ContainsKey(element))
             {
-                ElementsCounter.Add(element, 0);
+                ElementsCounter[element]++;
```

Comment:
Doesn't this fail if `ElementsCounter[element]` doesn't already exist in the dictionary because it is trying to do an operation on something that doesn't exist by the time ElementsCounter adds `element`?